### ISSUE TITLE
Summary values are not updated when Paste (Ctrl+V)

### ISSUE THREAD

**Issue by qwertwert** [nhn/tui.grid#1210]
**Describe the bug**

Summary values are not updated when Paste (Ctrl+V)

**To Reproduce**
1. Copy some value to Clip board (Ctrl+C)
2. Paste to Grid cell (Ctrl+V)
3. Cell value is pasted succesfully
4. But summary value at bottom (SUM) is not updated

**Expected behavior**
I expect update summary value

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser chrome
 - Version 86.0.4240.111(공식 빌드) (64비트)

**Additional context**

```
summary: {
	height: 40,
	position: 'bottom', // or 'top'
	columnContent: {
		rate: {
			template: function(valueMap) {
				return `합:<br>${valueMap.sum}%`;
			}
		}
	}
},
```
```
{
	header: '가중치',
	name: 'rate',
	editor: 'text',
	formatter({value}) {
		return String(`${value}`).includes('%') ? `${value}` : `${value}%`;
	},
	width: 50,
	validation: {
		dataType: 'number',
		min: 1,
		max: 100
	}
},
```

![image](https://user-images.githubusercontent.com/62734377/97252445-4854d680-184d-11eb-9d9b-b57eb51e6407.png)

**Comment by js87zz** [comment]
@qwertwert 
I checked the issue. 
When the issue will be resolved, I comment here.
Thank you!

**Comment by js87zz** [comment]
@qwertwert 
The bug has been fixed in `v4.15.3`.

**Comment by qwertwert** [comment]
Thank you for help. It works well!
